feat: add @kaname-png/plugin-statcord (#4)
sapphiredev/awesome-sapphire: README.md

## Contents

- [Awesome Sapphire ![Awesome](https://awesome.re)](#awesome-sapphire-)
  - [Contents](#contents)
  - [Plugins](#plugins)
  - [Utilities](#utilities)
  - [Reading](#reading)
  - [Miscellaneous](#miscellaneous)

>   - [Contribute](#contribute)

Reviewer: Revert

Reviewer: You still need to do this

## Plugins

- [@devtomio/plugin-botlist](https://github.com/devtomio/sapphire-plugin-botlist#readme): This plugin enables the
  integration of many bot lists such as [`top.gg`](https://top.gg), [`discordlabs`](https://discordlabs.org), and
  others.

> - [@kaname-png/plugin-statcord](https://github.com/kaname-png/neko-plugins/tree/main/packages/statcord): Plugin for
>   @sapphire/framework to send and manage bot stats with statcord.

Reviewer: ```suggestion
- [@kaname-png/plugin-statcord](https://github.com/kaname-png/neko-plugins/tree/main/packages/statcord): Plugin for [`@sapphire/framework`](https://github.com/sapphiredev/framework) to send and manage bot statistics on [statcord](https://statcord.com).
```

And run prettier afterwards
